src/scripts.js

```diff
@@ -59,29 +59,31 @@ function createUserRepo(wcUsersData) {
 
 function createIngredientsRepo(ingredientsData) {
   let ingredientsRepository = new IngredientsRepository(ingredientsData)
-  console.log('ingredients Repository', ingredientsRepository);
+  // console.log('ingredients Repository', ingredientsRepository);
   return ingredientsData
 }
 
 function createRecipesRepo(recipeData) {
-  console.log('recipeRepository', recipeData)
+  // console.log('recipeRepository', recipeData)
   let recipesRepository = new RecipesRepository(recipeData)
-  console.log('recipes Repository', recipesRepository);
+  // console.log('recipes Repository', recipesRepository);
```

Comment:
Looks like there are still a lot of `console.log`s hanging out in our code, we should probably go through and remove them if we're not still using them.